test-app/app_nrf52.c

```diff
@@ -43,6 +43,10 @@
 #define UART0_TXD_MAXCOUNT *((volatile uint32_t *)(UART0_BASE + 0x548))
 #define UART0_BAUDRATE     *((volatile uint32_t *)(UART0_BASE + 0x524))
 
+
+__attribute__((section(".rodata")))
+static const char extradata[1024 * 16] = "hi!";
```

Comment:
what's the purpose of this?

Reply:
Attempt to make the symbol appear in the app. This is actually redundant since this is the default section for const variables. The app will be >20KB only when DEBUG=1.